src/@types/common.ts

```diff
@@ -37,3 +37,20 @@ export type RouteTransition = {
   route: Route
   previousRoute: Route | null
 }
+
+type Component<T = {}> = {
+  name: string
+  url: string
+  meta: T
+}
+
+export type Components = {
+  'sgov-service': Component
```

Comment:
The typings for the components configuration would be less awkward if we opted for camelCase :-)

Reply:
Why not?